astroquery/alma/core.py

```diff
@@ -162,6 +162,7 @@
 def _gen_sql(payload):
     sql = 'select * from ivoa.obscore'
     where = ''
+    unused_payload = payload.copy()
```

Comment:
I wonder how many things can be shuffled into `**kwargs` in the first place, and therefore couldn't we actually address the route cause of this bug by spelling out those optional keyword arguments rather than hiding them in  `**kwargs` instead?